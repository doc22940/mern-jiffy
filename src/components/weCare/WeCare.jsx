import React from 'react';

import { WeCareContainer } from './WeCare.styles';

const WeCare = () => {
  return (
    <WeCareContainer
      href="https://twitter.com/urlDev"
      rel="noopener noreferrer"
      target="_blank"
    >
      <h3>Do you like the app?</h3>
      <img src={require('../../assets/V.gif')} alt="victory gif" />
      <img src={require('../../assets/heart.gif')} alt="heart gif" />
      <h3>
        <span>Jiffy</span> Cares
      </h3>
    </WeCareContainer>
  );
};

export default WeCare;
