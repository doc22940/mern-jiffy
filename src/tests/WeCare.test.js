import React from 'react';
import Enzyme, { mount } from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';

import WeCare from '../components/weCare/WeCare';

Enzyme.configure({ adapter: new EnzymeAdapter() });

test('Should render WeCare component', () => {
  const wrapper = mount(<WeCare />);
  expect(wrapper.debug()).toMatchSnapshot();
});
