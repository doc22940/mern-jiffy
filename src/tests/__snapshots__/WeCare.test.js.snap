// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render WeCare component 1`] = `
"<WeCare>
  <styled.a href=\\"https://twitter.com/urlDev\\" rel=\\"noopener noreferrer\\" target=\\"_blank\\">
    <a href=\\"https://twitter.com/urlDev\\" rel=\\"noopener noreferrer\\" target=\\"_blank\\" className=\\"sc-AxjAm eAwEQP\\">
      <h3>
        Do you like the app?
      </h3>
      <img src=\\"V.gif\\" alt=\\"victory gif\\" />
      <img src=\\"heart.gif\\" alt=\\"heart gif\\" />
      <h3>
        <span>
          Jiffy
        </span>
         Cares
      </h3>
    </a>
  </styled.a>
</WeCare>"
`;
